diff --git a/node_modules/notion-types/build/index.d.ts b/node_modules/notion-types/build/index.d.ts
index 1579c84..fc3ecc3 100644
--- a/node_modules/notion-types/build/index.d.ts
+++ b/node_modules/notion-types/build/index.d.ts
@@ -102,6 +102,7 @@ interface BaseContentBlock extends BaseBlock {
 interface BasePageBlock extends BaseBlock {
     properties?: {
         title: Decoration[];
+        Description?: Decoration[];
     };
     format: {
         page_full_width?: boolean;
